### PR TITLE
Mani UI

### PR DESCRIPTION
**Pull request by ManiAkasapu**
- Some CSS changes to connect an power buttons.
- Some useful icons added.

### CODE REVIEW

**Comment by ManiAkasapu** [comment]
@dexslab Please look at this and merge.

**Comment by dcyoung-dev** [comment]
@ManiAkasapu would you mind adding a screenshot to the PR description so we can see all your pretty pretties 🤩

File: package-lock.json
Comment (dcyoung-dev) [inline package-lock.json:1]: This file shouldn't be commited. We use `pnpm` which has it's own lockfile

**Comment by ManiAkasapu** [comment]
image

**Comment by dcyoung-dev** [comment]
Thanks @ManiAkasapu I've rebased and merged this branch 👌 deployment is rolling out just now and your changes will be live shortly 🎉

**Pull request merged**